.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,76 @@
+name: unit-tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+      - "open-release/*.master"
+  
+jobs:
+  run-tests:
+    if: (github.repository != 'openedx/edx-platform' && github.repository != 'edx/edx-platform-private') || (github.repository == 'openedx/edx-platform' && (startsWith(github.base_ref, 'open-release') == true))
+    runs-on: [ ubuntu-20.04 ]
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8]
```

Comment:
Wouldn't it make more sense to bind the specific Python version to the `edx-platform` release we're using for the test? No named releases officially support more than one version.

Reply:
Sure. Both nutmeg.2 & master were using python 3.8. So removed the matrix python-version and used directly in python installer action.